calendarpal/build.gradle

```diff
@@ -41,14 +40,16 @@ android {
 }
 
 dependencies {
-    compile 'com.android.support:support-annotations:' + SUPPORT_LIBRARY_VERSION
-    compile project(':contentpal');
-    compile "org.dmfs:rfc5545-datetime:0.2.4"
-    compile "org.dmfs:lib-recur:0.10"
+    api project(':contentpal')
 
-    testCompile 'junit:junit:' + JUNIT_VERSION
-    testCompile 'org.mockito:mockito-core:' + MOCKITO_VERSION
-    testCompile 'org.robolectric:robolectric:' + ROBOLECTRIC_VERSION
-    testCompile 'org.dmfs:jems-testing:' + JEMS_VERSION
-    testCompile project(':contentpal-testing')
+    implementation 'com.android.support:support-annotations:' + SUPPORT_LIBRARY_VERSION
+    implementation 'org.dmfs:rfc5545-datetime:' + RFC5545_DATETIME_VERSION
+    implementation 'org.dmfs:lib-recur:0.10'
+    implementation 'org.dmfs:jems:' + JEMS_VERSION
+
+    testImplementation project(':contentpal-testing')
+    testImplementation 'junit:junit:' + JUNIT_VERSION
+    testImplementation 'org.mockito:mockito-core:' + MOCKITO_VERSION
+    testImplementation 'org.robolectric:robolectric:' + ROBOLECTRIC_VERSION
+    testImplementation 'org.dmfs:jems-testing:' + JEMS_VERSION
```

Comment:
this is not necessary when it's already an `implementation` dependency

Reply:
Which one do you mean here? If it was `jems` please note that that's `jems-testing` here.

Reply:
Oh, right. I missed that. It's ok then.